platforms/allwinner-d1/boards/lichee-rv/src/main.rs

```diff
@@ -0,0 +1,55 @@
+#![no_std]
+#![no_main]
+
+use core::{panic::PanicInfo, time::Duration};
+use drivers::Ram;
+use kernel::{Kernel, KernelSettings};
+use embedded_hal::blocking::delay::DelayMs;
+
+const HEAP_SIZE: usize = 384 * 1024 * 1024;
```

Comment:
would be cool if, eventually, there was a nice way to keep this in sync with the linker script...we can figure that out later though